lib/widgets/content.dart

```diff
@@ -107,6 +109,27 @@ class BlockContentList extends StatelessWidget {
   }
 }
 
+/// A Divider matching the style found in:
+/// [web/styles/rendered_markdown.css](https://github.com/zulip/zulip/blob/main/web/styles/rendered_markdown.css)
+class ThematicBreak extends StatelessWidget {
```

Comment:
I think we can write this widget without a dartdoc, like the other content widgets. A person reading the code will assume that our UI elements correspond with the web app's UI elements. It can be useful to refer to specific web-app code in places where we intentionally deviate from web (for example to adapt to the mobile environment, such as in the absence of hover interactions). But that's not what we're doing here.